pages/company/components/Story.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-unescaped-entities */
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { FC } from "react";
+import styles from "../company.module.scss";
+import { story4 } from "assets/png";
+import Image from "next/image";
+
+const story = [
+	{ title: "	✓ Founded in 2014" },
+	{ title: "	✓ Community of 125K+" },
+	{ title: "	✓ $33M in funding" },
+	{ title: "	✓ 500+ global meetups" },
+];
+export const Story: FC = () => {
+	return (
+		<div className={styles.story}>
+			<Typography
+				variant="h4"
+				sx={theme => ({
+					[theme.breakpoints.down("sm")]: {
+						textAlign: "center",
+					},
+				})}
+			>
+				<Typography variant="h5" mb={3}>
+					OUR STORY
+				</Typography>
+				We’re going to become <br /> partners for long run.
+			</Typography>
+			<Box>
+				<Typography maxWidth={559} textAlign="center">
+					Lorem ipsum dolor sit amet consectetur, adipisicing elit. Et odit
+					quibusdam voluptates dolores pariatur porro ipsum nisi illo,
+					cupiditate iste, asperiores totam exercitationem?{" "}
+				</Typography>{" "}
+				<br />
+				<Box
+					className={styles.cards}
+					sx={theme => ({
+						[theme.breakpoints.down("sm")]: {
+							gridTemplateAreas: '"a" "b" "c" "d" !important',
+							width: "100%",
+						},
+					})}
+				>
+					{story.map((feat, i) => {
+						const textAlign = i % 2 === 0 ? "left" : "right";
+						return (
+							<Box
+								className={styles[`card${i + 1}`]}
+								key={`card${i + 1}`}
+								sx={theme => ({
+									[theme.breakpoints.down("sm")]: {
+										textAlign,
+										width: "100% !important",
+										px: "7vw",
+									},
+								})}
+							>
+								<Typography variant="subtitle1">{feat.title}</Typography>
+							</Box>
+						);
+					})}
+				</Box>
+			</Box>
+			<Box mt={5} display="flex">
+				<div>
+					<Image src={story4} alt="" />
```

Comment:
![image](https://user-images.githubusercontent.com/60546840/192575342-36650f8d-b4e1-40b6-a5ca-792024ef99ff.png)

1. The whole parent box is not responsive for smaller screens. so apply these styles:
- first put `max-width: 600px`to this image
- add `flex-wrap: wrap` to tha parent

2. add margins to the text. (its touching the boundaries)

Reply:
Do all these changes... rest LGTM

Reply:
how to give max-width to the image

Reply:
You cant.

For that I earlier created our own `Image` component. That custom `Image` component's `wrapstyle` prop add these kind of styles. (Look at our custom `Image` component code.. its a global component. You will get an idea)